docs/tutorial/tutorial.md

```diff
@@ -200,8 +201,7 @@ handleClick(i) {
   this.setState({squares: squares});
 }
 ```
-
-We call `.slice()` to copy the `squares` array instead of mutating the existing array. Jump ahead a [section](/react/tutorial/tutorial.html#why-immutability-is-important) to learn why immutability is important.
+It's popular to name handler props starting with `on` and their implementations with `handle`. We call `.slice()` to copy the `squares` array instead of mutating the existing array. Jump ahead a [section](/react/tutorial/tutorial.html#why-immutability-is-important) to learn why immutability is important.
```

Comment:
Let's add a little more context to this first sentence as well:

> All of React's event handler props start with `on`, for example, `onClick` and `onFocus`. It's a common pattern to name the functions passed to those props starting with `handle`, such as `handleCick` for `onClick` or `handleFocus` for `onFocus`. It's a useful pattern for being consistent with how event handlers are defined, but it's not required.

It's a little more verbose, but I think the inline example and clarification that it's not required are useful.